Evaluate-ModelarDB-Compression/main.py

```diff
@@ -342,12 +342,17 @@ def send_sigint_to_process(process):
                 try:
                     test_data_field_column = test_data.column(field_column_name)
                 except:
-                    # Spaces in the name may have been replaced by underscores.
-                    field_column_name_with_space = field_column_name.replace("_", " ")
-                    test_data_field_column = test_data.column(
-                        field_column_name_with_space
-                    )
-
+                    if field_column_name.__contains__("_"):
```

Comment:
Generally Python's magic methods should not be called directly as they exist to allow user-defined types to work with Python's built-in functionality such as keywords and operators. Since [`__contains__`](https://docs.python.org/3/reference/datamodel.html#object.__contains__) is called by the [membership test operators](https://docs.python.org/3/reference/expressions.html#membership-test-details) they should be  used instead of the if statement is necessary.

Reply:
Removed if statement to handle both cases together.